solidity/contracts/Rewards.sol

```diff
@@ -79,29 +88,21 @@ contract Rewards {
     // `totalRewards.sub(paidOutRewards)`
     uint256 public paidOutRewards;
 
-    // Length of one interval in seconds (timestamp diff).
-    uint256 public constant termLength = 100; // TODO define
     // Timestamp of first interval beginning.
     // Interval 0 covers everything before `firstIntervalStart`
     // and the first `termLength` after `firstIntervalStart`.
     // This value should be set
```

Comment:
I plan to change this in another commit. We should be able to fund the contract after this date.

Reply:
> We should be able to fund the contract after this date.

But if the `firstIntervalStart` is Sep 24, term length is 30 days and according to "This value should be set so that `firstIntervalStart + termLength` is in the future, so the contract can be funded before intervals can be allocated." then doesn't it mean that we'll have time till Oct 24th to fund the contract?

Reply:
> But if the firstIntervalStart is Sep 24, term length is 30 days and according to "This value should be set so that firstIntervalStart + termLength is in the future, so the contract can be funded before intervals can be allocated." then doesn't it mean that we'll have time till Oct 24th to fund the contract?

It does and we are lucky if we deploy before Oct 24, but if we do it Oct 25 then we need that mechanism 😉